commands/doctor.go

```diff
@@ -206,5 +206,10 @@ func (cmd *Doctor) Run(c *cli.Context) error {
 		}
 	}
 
+	// 7. Check for availability of a rig upgrade
+	cmd.out.Spin("Checking for available rig updates...")
+	if msg := util.CheckForRigUpdate(c.App.Version); msg != "" {
+		cmd.out.Info(msg)
+	}
```

Comment:
I thought it was nice that doctor would say that `rig` was up to date. You can leave that message, if you'd like